osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -137,11 +139,133 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
                                         Shadow = true,
                                     },
                                 }
-                            }
+                            },
+                            new Container
+                            {
+                                Margin = new MarginPadding { Top = 20 },
+                                AutoSizeAxes = Axes.Both,
+                                Children = new Drawable[]
+                                {
+                                    new Container

```

Comment:
There is lots of redundancy in the Container/SpriteText pairs you are adding to the hierarchy for every piece of information. Encapsulate that in a delegate or function that generates these. Wrapping the two in an additional container might make sense. Then you can do something like:

`Children = new []
{
    infoLabel(FontAwesome.fa_clock_o, length),
    infoLabel(FontAwesome.fa_circle_o, bpm),
    infoLabel(FontAwesome.fa_whatever, whatever),
    ....
}
`